src/testers.jl

```diff
@@ -44,6 +39,29 @@ function _make_fdm_call(fdm, f, ȳ, xs, ignores)
         @assert length(callargs) == length(xs)
         return f(callargs...)
     end
+    return fnew
+end
+
+@deprecate _make_fdm_call(fdm, f, ȳ, xs, ignores) _make_j′vp_call(fdm, f, ȳ, xs, ignores)
```

Comment:
Why are we deprecating this?
Its is a purely internal function.

ChainRules.jl doesn't use it it its tests, I don't think anyone does

Reply:
Good point, I had remembered seeing someone (maybe me) use it elsewhere, but maybe that code was deleted; no one else on GitHub uses it now.